### ISSUE TITLE
`export * as ns` false positive on import/named rule

### ISSUE THREAD

**Issue by mAAdhaTTah** [benmosher/eslint-plugin-import#1883]
```js
// module1.js
export const hello = 'world';

// module2.js
export * as mod1 from './module1'

// module3.js
import { mod1 } from './module2'; // <-- getting "mod1 not found", triggered by `import/named` rule.
```

I can create a repro case if needed.

**Comment by ljharb** [comment]
Thanks for the report! If you're not able to PR a fix, a PR with the failing test case would be stellar <3

**Comment by mAAdhaTTah** [comment]
#1885 for failing tests. I could probs fix with some guidance.

**Comment by rkulinski** [comment]
Still an issue :(

**Comment by ljharb** [comment]
This `import/named` false positive was fixed back in v2.24.1 by 3ff4d772 (`[Fix] named, namespace: properly set reexports on export * as … from`) - the re-exported name is registered correctly now. Closing as resolved; please reopen with a repro if you still hit it on a current version.
